apps/mobile/src/components/TxInfo/TxInfo.tsx

```diff
@@ -140,10 +140,4 @@ function TxInfoComponent({ tx, onPress, ...rest }: TxInfoProps) {
   return <></>
 }
 
-export const TxInfo = React.memo(TxInfoComponent, (prevProps, nextProps) => {
-  return (
-    prevProps.tx.txHash === nextProps.tx.txHash &&
-    prevProps.bordered === nextProps.bordered &&
-    prevProps.inQueue === nextProps.inQueue
```

Comment:
Why did we need this in the first place? I thought it is responsible for rerendering the component if any of these props change but not others.